terraform/modules/avl-consumer-subscriptions/main.tf

```diff
@@ -192,6 +192,37 @@ module "avl_consumer_unsubscriber" {
   }
 }
 
+
+module "avl_consumer_subscriptions" {
+  source = "../shared/lambda-function"
+
+  environment   = var.environment
+  function_name = "integrated-data-avl-consumer-subscriptions"
+  zip_path      = "${path.module}/../../../src/functions/dist/avl-consumer-subscriptions.zip"
+  handler       = "index.handler"
+  memory        = 256
+  runtime       = "nodejs20.x"
+  timeout       = 60
+
+  permissions = [
+    {
+      Action = [
+        "dynamodb:Scan"
```

Comment:
Think this lambda will also need permissions to query dynamo in the case we want to return a single subscription